spec.html

```diff
@@ -23791,7 +23811,7 @@ <h1>Runtime Semantics: Evaluation</h1>
         1. Return ~empty~.
       </emu-alg>
       <emu-note>
-        <p>An alternative semantics is provided in <emu-xref href="#sec-block-level-function-declarations-web-legacy-compatibility-semantics"></emu-xref>.</p>
+        <p>An alternative semantics is provided by <emu-xref href="#sec-block-level-function-declarations-web-legacy-compatibility-semantics" title></emu-xref> at step <emu-xref href="#step-functiondeclarationinstantiation-alt-funcdecl-eval"></emu-xref> of FunctionDeclarationInstantiation, step <emu-xref href="#step-globaldeclarationinstantiation-alt-funcdecl-eval"></emu-xref> of GlobalDeclarationInstantiation, and step <emu-xref href="#step-evaldeclarationinstantiation-alt-funcdecl-eval"></emu-xref> of EvalDeclarationInstantiation.</p>
```

Comment:
Fine for now but I'd like to find some way to inline these as well in a followup. Having a runtime algorithm step say "and then when you perform this other step, do this other thing" is crazy.